Sources/Overload/OvEditor/src/OvEditor/Core/GizmoBehaviour.cpp

```diff
@@ -13,6 +13,15 @@ float SnapValue(float p_value, float p_step)
 	return p_value - std::fmod(p_value, p_step);
 }
 
+OvMaths::FVector3 SnapValue(OvMaths::FVector3 p_value, float p_step)
+{
+	OvMaths::FVector3 result;
+	result.x = floor(p_value.x / p_step) * p_step;
+	result.y = floor(p_value.y / p_step) * p_step;
+	result.z = floor(p_value.z / p_step) * p_step;
```

Comment:
`std::round` seems to provide more consistent result, as otherwise using `floor` makes moving in one direction "easier" than in the other.

For instance, with a snap value of 1, when starting to drag the gizmo, if the offset on a single axis is +0.1, `floor` will keep the object in place unless the positive offset is `>= 1`, while any negative offset (let's say -0.1) would snap the object to `-1`, resulting in unbalanced translation.

```suggestion
	result.x = std::round(p_value.x / p_step) * p_step;
	result.y = std::round(p_value.y / p_step) * p_step;
	result.z = std::round(p_value.z / p_step) * p_step;
```

Reply:
Indeed I did not noticed that at first sight but totally correct!